can/ref/ref.js

```diff
@@ -385,8 +388,8 @@ var makeRef = function(connection){
 	 * @signature `ref.serialize`
 	 * @return {string} id the id of the referenced instance
 	 */
-	Ref.prototype.serialize = function() {
-		return this[idProp];
+	Ref.prototype.serialize = Ref.prototype[canSymbol.for("can.serialize")] = function() {
+		return canReflect.serialize(this[idProp], CIDMap);
```

Comment:
Is there a good reason to provide a Map here? Seems to me like it would only be good for circular reference, which shouldn't happen here.